packages/card/src/private/wrappers.ts

```diff
@@ -10,28 +11,10 @@ type __CardProps = UiCardProps &
 
 export const CardWrapper = styled.div<__CardProps>`
   ${(props) => `
-    color: ${getThemeColor(props.customTheme, props.selectedTheme, ColorCategories.fonts, ColorTokens.token_100)};
-    background-color: ${getThemeColor(
-      props.customTheme,
-      props.selectedTheme,
-      ColorCategories.backgrounds,
-      ColorTokens.token_100
-    )};
-
-    border: 1px solid ${getThemeColor(
-      props.customTheme,
-      props.selectedTheme,
-      ColorCategories.backgrounds,
-      ColorTokens.token_100
-    )};
+    ${getThemeStyling(props.customTheme, props.selectedTheme, themeMapper)}
```

Comment:
This function will output basically the same styles that we were setting manually but will do it automatically based on a mapper file here: https://github.com/inavac182/uireact/pull/60/files#diff-4869622ca559204aadc34ab27d0429ad8fee6035fbf484005a0c997ece7e31e0R3